src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -789,6 +789,13 @@ because xbuild doesn't support framework reference assemblies.
 				Condition="'$(DxJarPath)' == ''"
 		/>
 	</CreateProperty>
+
+    <CreateProperty Value="--dex --no-strict">
+        <Output TaskParameter="Value" PropertyName="DxExtraArguments"
+                Condition="'$(DxExtraArguments)' == ''"
+        />
+    </CreateProperty>
+
```

Comment:
@atsushieno looks like this bit is using spaces rather than tabs. I think @jonpryor likes to keep the formatting similar to the sourounding code.

Reply:
fixed!